egs/cifar/v1/image/ocr/yomdle/create_download_dir.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
```

Comment:
I'm not sure if this `yomdle` directory should be a subdirectory of `image/ocr` because I guess the shared scripts should not be dataset-specific. Maybe this could be a subdirectory of `egs/yomdle_tamil/v1/local` and other yomdle recipes can point to it?